src/Utils/AST.php

```diff
@@ -588,7 +588,7 @@ public static function typeFromAST(Schema $schema, $inputTypeNode)
      *
      * @param string $operationName
      *
-     * @return bool
+     * @return bool|string
```

Comment:
can move to return type

Reply:
How so? Does PHP support union types?

Reply:
Ah, my bad. seen `|null`

[but soon!](https://github.com/php/php-rfcs/pull/1)